Gems/Atom/RHI/Code/Include/Atom/RHI/DrawItem.h

```diff
@@ -134,7 +134,7 @@ namespace AZ::RHI
             {
                 bool m_enabled : 1;    // Whether the Draw Item should render
             };
-            uint8_t m_allFlags;
+            uint8_t m_allFlags = 1;
```

Comment:
I think it's better to add m_enabled = true in constructor if `bool m_enabled : 1 = 1;` is not supported.
The union was added to have the potential of more than one flags. 
The m_allFlags = 1; here you have is to intent to set m_enabled = true which is implicit. 
If people add a flag before m_enabled in the future which could easily break the intention.

Reply:
Another option would be use m_disabled and set m_allFlags = 0; by default